job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -94,7 +94,7 @@ pipeline{
             steps {
                 build job: 'kogito-tooling.nightly/0.13.0-prerelease', propagate: false, wait: true, parameters: [
                         [\$class: 'StringParameterValue', name: 'DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-7.12/content-compressed'],
```

Comment:
We could derive the repository name in a similar way as UMB version, wdyt?

Reply:
And maybe have a constant for the product branch (7.59.x), so there would be no need to update this part of the pipeline.

Reply:
is this actually kogito-tooling needed here? we can remove/comment it out this step

Reply:
and if we change and unify on string without dot it could be even less variable